topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -330,57 +329,14 @@ def _decl_winograd(cfg, data, kernel, strides, padding, dilation, layout, out_dt
     HPAD, WPAD, _, _ = get_pad_tuple(padding, kernel)
 
     assert layout == 'NCHW'
-    assert KH == 3 and KW == 3 and HPAD == 1 and WPAD == 1 and HSTR == 1 and WSTR == 1
+    assert KH == KW and HSTR == 1 and WSTR == 1
```

Comment:
Add test case for this change?

Reply:
Other parts lgtm.

Reply:
@kevinthesun ,

- Added 5x5, 7x7 [testcases](https://github.com/dmlc/tvm/pull/3553/files#diff-4d45eede1cdc31952846a1e64533f22cR108) (cuda only)
- Constrainted [arm/mali](https://github.com/dmlc/tvm/pull/3553/files#diff-7ac4fe4f71257850a5471c9c63647bb0R332) to 3x3 only (until fix it in other PR)